catalogue/webapp/components/ItemRequestModal/ItemRequestModal.tsx

```diff
@@ -382,16 +386,18 @@ const ConfirmedDialog: FC<ConfirmedDialogProps> = ({ currentHoldNumber }) => (
 
 type ErrorDialogProps = {
   setIsActive: (value: boolean) => void;
+  errorMessage: string | undefined;
 };
 
-const ErrorDialog: FC<ErrorDialogProps> = ({ setIsActive }) => (
+const ErrorDialog: FC<ErrorDialogProps> = ({ setIsActive, errorMessage }) => (
   <>
     <Header>
       <span className={`h2`}>Request failed</span>
     </Header>
     <p className="no-margin">
       {/* TODO: get error code and construct appropriate message from response - see #6916 */}
-      There was a problem requesting this item. Please try again.
+      {errorMessage ||
+        'There was a problem requesting this item. Please try again.'}
```

Comment:
While we're here, maybe we move this default error message to the microcopy file?

Reply:
Great shout!